com.visualstudio.code.yaml

```diff
@@ -40,31 +41,6 @@ cleanup:
   - /share/gtk-doc
   - '*.la'
 modules:
-  # Provide libsecret without gcrypt, forcing it to use the D-Bus backend
```

Comment:
why this change?

Reply:
Ah, sorry. I forgot to add it to the PR description, it's only described in the commit message 8fed65782adb84fb0e9f5ab38f4d401e095f7340:

> Use libsecret from base app
>
> The branch 25.08 of the base app `org.electronjs.Electron2.BaseApp` already
provides libsecret with the crypto backend disabled.
>
> Ref: https://github.com/flathub/org.electronjs.Electron2.BaseApp/commit/24e075eb32f6d638874f7d5533337350a3eb0a0d

So it's only a minor cleanup. I made the same change already in the signal app without any issues, but I can remove it from this PR if you prefer.